src/sentry/static/sentry/app/components/actions/ignore.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {t} from '../../locale';
+import MenuItem from '../menuItem';
+import DropdownLink from '../dropdownLink';
+import Duration from '../duration';
+import CustomIgnoreCountModal from '../customIgnoreCountModal';
+import CustomIgnoreDurationModal from '../customIgnoreDurationModal';
+import ActionLink from './actionLink';
+
+export default class IgnoreActions extends React.Component {
+  static propTypes = {
+    isIgnored: PropTypes.bool,
+    onUpdate: PropTypes.func.isRequired,
+    disabled: PropTypes.bool,
+    shouldConfirm: PropTypes.bool,
+    confirmMessage: PropTypes.node,
+    confirmLabel: PropTypes.string,
+  };
+
+  static defaultProps = {
+    isIgnored: false,
+    confirmLabel: 'Ignore',
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {modal: false};
+  }
+  getIgnoreDurations() {
+    return [30, 120, 360, 60 * 24, 60 * 24 * 7];
+  }
+
+  getIgnoreCounts() {
+    return [100, 1000, 10000, 100000];
```

Comment:
random thing, but lets add `10` here now? i think it would be the most used option :P